src/pathology-report/ReportComponent.tsx

```diff
@@ -1,40 +1,476 @@
+        (encList.middle_name &&
+          encList.middle_name
+            .toLowerCase()
+            .includes(patientName.toLowerCase())) ||
+        (encList.given_name &&
+          encList.given_name.toLowerCase().includes(patientName.toLowerCase()))
+    );
+  // const rows = filteredEncList.map(
+  //   (encounterInfo) => {
+  //       return {
+  //           // patientLink: <a href={`/openmrs/patientDashboard.form?patientId=${encounterInfo.personId}`} >Link</a>,
+  //           ...
+
+  var rows = filteredEncList.map((encounterInfo) => {
+    return {
+      // rows.push({
+      // patientLink: <a href={`/openmrs/patientDashboard.form?patientId=${encounterInfo.personId}`} >Link</a>,
+      id: `${encounterInfo.encounterId}`,
+      patientNames: (
+        <a
+          href={`/openmrs/patientDashboard.form?patientId=${encounterInfo.personId}`}
+        >
+          {encounterInfo.family_name ? encounterInfo.family_name : ""}{" "}
+          {encounterInfo.given_name ? encounterInfo.given_name : ""}{" "}
+          {encounterInfo.middle_name ? encounterInfo.middle_name : ""}{" "}
+        </a>
+      ),
+      pathologyRequest: (
+        <a
+          href={`/openmrs/module/htmlformentry/htmlFormEntry.form?encounterId=${encounterInfo.encounterId}&mode=EDIT`}
+        >
+          {" "}
+          Link{" "}
+        </a>
+      ),
+      SendingHospital: `${encounterInfo.patientHealthCenter}`,
+      phoneNumber: `${
+        encounterInfo.patientPhoneNumber ? encounterInfo.patientPhoneNumber : ""
+      }`,
+      sampleStatus: (
+        <select
+          onChange={(e) =>
+            sampleStatusChange(
+              e.target.value && JSON.parse(e.target.value),
+              encounterInfo
+            )
+          }
+        >
+          <option value=""></option>
+          {sampleStatusResults.map((ans) => (
+            <option
+              value={`{"uuid": "${ans.uuid}","display": "${ans.display}"}`}
+              key={ans.uuid}
+              selected={
+                encounterInfo.sampleStatusObs
+                  ? encounterInfo.sampleStatusObs === ans.display && true
+                  : false
+              }
+            >
+              {ans.display}
+            </option>
+          ))}
+        </select>
+      ),
+      dateOfRequest: `${encounterInfo.encounterDatetime}`,
+      referralStatus: (
+        <select
+          onChange={(e) =>
+            referralStatusChange(
+              e.target.value && JSON.parse(e.target.value),
+              encounterInfo
+            )
+          }
+        >
+          <option value=""></option>
+          {referralStatusResults.map((ans) => (
+            <option
+              value={`{"uuid": "${ans.uuid}","display": "${ans.display}"}`}
+              key={ans.uuid}
+              selected={
+                encounterInfo.referralStatusObs
+                  ? encounterInfo.referralStatusObs === ans.display && true
+                  : false
```

Comment:
```suggestion
                encounterInfo.referralStatusObs && encounterInfo.referralStatusObs === ans.display
```

Wrap it in a `Boolean()` if needed.